ffi/diplomat/src/locale.rs

```diff
@@ -240,6 +230,108 @@ pub mod ffi {
             let other = other.as_bytes(); // #2520
             self.0.strict_cmp(other).into()
         }
+
+        //============ CONSTANTS ============
+
+        /// Construct an [`ICU4XLocale`] for the `ar` locale.
+        pub fn create_ar() -> Box<ICU4XLocale> {
+            Box::new(ICU4XLocale(icu_locid::locale!("ar")))
+        }
+
+        /// Construct an [`ICU4XLocale`] for the `bn` locale.
+        pub fn create_bn() -> Box<ICU4XLocale> {
+            Box::new(ICU4XLocale(icu_locid::locale!("bn")))
+        }
+
+        /// Construct an [`ICU4XLocale`] for the `de` locale.
+        pub fn create_de() -> Box<ICU4XLocale> {
+            Box::new(ICU4XLocale(icu_locid::locale!("de")))
+        }
+
+        /// Construct an [`ICU4XLocale`] for the `en` locale.
+        pub fn create_en() -> Box<ICU4XLocale> {
+            Box::new(ICU4XLocale(icu_locid::locale!("en")))
+        }
+
+        /// Construct an [`ICU4XLocale`] for the `en-GB` locale.
+        pub fn create_en_gb() -> Box<ICU4XLocale> {
+            Box::new(ICU4XLocale(icu_locid::locale!("en-GB")))
+        }
+
+        /// Construct an [`ICU4XLocale`] for the `es` locale.
+        pub fn create_es() -> Box<ICU4XLocale> {
+            Box::new(ICU4XLocale(icu_locid::locale!("es")))
+        }
+
+        /// Construct an [`ICU4XLocale`] for the `es-419` locale.
+        pub fn create_es_419() -> Box<ICU4XLocale> {
+            Box::new(ICU4XLocale(icu_locid::locale!("es-419")))
+        }
```

Comment:
Thought: I wonder if in new code we should favor `en-001` instead of `en-GB`, or `es-AR` (or whichever is the most populated 419 country) instead of `es-419`

Reply:
@macchiati Should we favor `en-001` or `en-GB`?